src/main/java/net/spy/memcached/ArcusReplNodeAddress.java

```diff
@@ -87,6 +87,48 @@ private static List<InetSocketAddress> parseNodeNames(String s) {
     return addrs;
   }
 
+
+  private static List<InetSocketAddress> parseNodeNames(List<String> s) {
+    List<InetSocketAddress> addrs = new ArrayList<InetSocketAddress>();
+
+    for (String node : s) {
+      String[] temp = node.split("\\^");
+      String group = temp[0];
+      boolean master = temp[1].equals("M");
+      String ipport = temp[2];
+      // We may throw null pointer exception if the string has
+      // an unexpected format.  Abort the whole method instead of
+      // trying to ignore malformed strings.
+      // Is this the right behavior?  FIXME
+
+      addrs.add(ArcusReplNodeAddress.create(group, master, ipport));
+    }
+    return addrs;
+  }
```

Comment:
아래 메소드 사용하면, 두 군데서 활용할 수 있습니다.
```java
  private static InetSocketAddress parseNodeName(String node) {
    String[] temp = node.split("\\^");
    String group = temp[0];
    boolean master = temp[1].equals("M");
    String ipport = temp[2];
    // We may throw null pointer exception if the string has
    // an unexpected format.  Abort the whole method instead of
    // trying to ignore malformed strings.
    // Is this the right behavior?  FIXME
    return ArcusReplNodeAddress.create(group, master, ipport);
  }
```